torch/_subclasses/fake_tensor.py

```diff
@@ -1424,6 +1424,8 @@ def maybe_run_unsafe_fallback(error=None):
                 r = func(*args, **kwargs)
         except NotImplementedError as not_implemented_error:
             return maybe_run_unsafe_fallback(not_implemented_error)
+        except RuntimeError as e:
+            raise UnsupportedOperatorException(str(e)) from e
```

Comment:
Do you have a smaller repro that demonstrates the error?

I don't think we want to do this: this is converting all RuntimeErrors into graph breaks. If there is a RuntimeError because someone implemented the FakeTensor kernel incorrectly, we want to know about it so that we can fix it.